server/models/jobRole.model.js

```diff
@@ -8,18 +8,26 @@ const jobRole = new Schema({
         unique: true,
         required: true
```

Comment:
Optionally we could ad status with types active, expired, etc. PS Note- this can be used on models like courses, university, user etc. Currently we have isBlocked, maybe we can use status instead.